polars/polars-lazy/polars-plan/src/logical_plan/optimizer/type_coercion/mod.rs

```diff
@@ -514,28 +514,3 @@ fn early_escape(type_self: &DataType, type_other: &DataType) -> Option<()> {
         Some(())
     }
 }
-
-#[cfg(test)]
-#[cfg(feature = "dtype-categorical")]
-mod test {
-    use polars_core::prelude::*;
-
-    use super::*;
-    use crate::prelude::*;
-
-    #[test]
-    fn test_categorical_utf8() {
```

Comment:
This test did not compile - looks like function signatures are out of date. Couldn't figure out how to fix the test - so removing it for now.

Reply:
Can we try to move this test to `polars-lazy` or integration tests and make it work? The thing it tests, I don't want to break.

Reply:
Sure, I'll give it a shot!

Reply:
Will do that in a separate PR - merging the doc examples separately now.